### PR TITLE
Feature/cambios q rdesatendido

### PR DESCRIPTION
**Pull request by juanseferrari2**
## Description
Realizados algunos cambios en la documentacion del modelo desatendido.
Sacamos algunos valores que generaban errores en la App. Ordenamos las nomenclaturas.
Agregamos tambien un aviso de la API que no permite generar cajas sin sucursales asignadas. Dicen de agregarlo al Changelog? o como un aviso?

### Issue involved
Fixes #<issue>

### Checklist:
Before sending your contribution, please specify the following: 

- [ ] This request modifies code snippets. 
- [ ] The contribution aligns with the information stated in the repository wiki.

- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [ ] Contribution includes new features. 
- [ ] New features were communicated in changelog.
- [ ] Requires translations.

### CODE REVIEW

**Review by lamaralmp — APPROVED** [review]

**Comment by lamaralmp** [comment]
@juanseferrari2 sumé un par de ajustes de markdown y traducciones. Podrías validarlos?

File: guides/in-person-payments/qr-code/stores-pos.es.md [hunk 1]
```diff
@@ -118,6 +118,12 @@ https://api.mercadopago.com/pos \
 
 Puedes obtener más información en la [Referencias de API](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/reference/pos/_pos/post).
 
+> WARNING 
+> 
+> IMPORTANTE
+> 
+> A partir del 20 de Abril del 2021, no se podrán crear cajas sin una sucursal asignada.
```
Comment (pamalv7) [inline guides/in-person-payments/qr-code/stores-pos.es.md:125]:
```suggestion
> A partir del 20 de abril de 2021, no se podrán crear cajas sin una sucursal asignada. 
```

File: guides/in-person-payments/qr-code/stores-pos.en.md [hunk 2]
```diff
@@ -118,6 +118,12 @@ https://api.mercadopago.com/pos \
 
 Learn more with our [API Reference](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/reference/pos/_pos/post).
 
+> WARNING 
+> 
+> IMPORTANT
+> 
+> As of April 20, 2021, Point of sales can not be created without an assigned store.
```
Comment (pamalv7) [inline guides/in-person-payments/qr-code/stores-pos.en.md:125]:
```suggestion
> After April 20th, 2021, Point of Sales can not be created without an assigned store.
```

File: guides/in-person-payments/qr-code/stores-pos.en.md [hunk 3]
```diff
@@ -118,6 +118,12 @@ https://api.mercadopago.com/pos \
 
 Learn more with our [API Reference](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/reference/pos/_pos/post).
 
+> WARNING 
+> 
+> IMPORTANT
```
Comment (pamalv7) [inline guides/in-person-payments/qr-code/stores-pos.en.md:123]:
```suggestion
> Important
```

File: guides/in-person-payments/qr-code/stores-pos.es.md [hunk 4]
```diff
@@ -118,6 +118,12 @@ https://api.mercadopago.com/pos \
 
 Puedes obtener más información en la [Referencias de API](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/reference/pos/_pos/post).
 
+> WARNING 
+> 
+> IMPORTANTE
```
Comment (pamalv7) [inline guides/in-person-payments/qr-code/stores-pos.es.md:123]:
```suggestion
> Importante
```

File: guides/in-person-payments/qr-code/stores-pos.pt.md [hunk 5]
```diff
@@ -118,6 +118,12 @@ https://api.mercadopago.com/pos \
 
 Você pode obter mais informações em [Referências de API](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/reference/pos/_pos/post).
 
+> WARNING 
+> 
+> IMPORTANTE
```
Comment (pamalv7) [inline guides/in-person-payments/qr-code/stores-pos.pt.md:123]:
```suggestion
> Importante
```

**Review by pamalv7 — CHANGES_REQUESTED** [review]

**Review by pamalv7 — APPROVED** [review]

**Pull request merged**